src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Email/Blocks/email_configuration.html.twig

```diff
@@ -38,6 +38,19 @@
           <div class="js-smtp-configuration{% if emailConfigurationForm.mail_method.vars.value != smtpMailMethod %} d-none{% endif %}">
             {{ form_widget(emailConfigurationForm.smtp_config) }}
           </div>
+          {{ form_row(emailConfigurationForm.mail_type) }}
+          {{ form_row(emailConfigurationForm.log_emails) }}
+          <div class="alert alert-info" role="alert">
+            <p class="alert-text">
+              {{ 'Fill in the fields below to set up DKIM signing of outgoing emails. This will reduce the likelihood of your emails being marked as spam.'|trans({}, 'Admin.Advparameters.Help') }}
```

Comment:
Is there a reason why the paragraph is divided into 3 different strings? 
If not, I think it would be great to have the 3 strings together, like this:
![image](https://user-images.githubusercontent.com/70583503/111785015-5c60a980-88bc-11eb-9e11-26bf4afaab79.png)
This would make the work of translators easier.